add the planet decred website files created in hugo.

rickmort opened the pull request:
•This PR adds the initial files of the planet decred website.

•Everything was built in Hugo to implement speed to the website and easy future maintenance.

Commit:
add web files

Commit:
add files

Commit:
eliminate licenses

Commit:
render website

Commit:
Update config.toml

Commit:
render website

Commit:
add

Commit:
add netlify.toml config

Commit:
for environment

Commit:
update url blog to test in netlify

xaur commented on docs/about/index.html:
```diff
@@ -0,0 +1,118 @@
+
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather:300,700%7CSource+Code+Pro:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css" integrity="sha384-KA6wR/X5RY4zFAHpv/CnoG2UW1uogYfdnP67Uv7eULvTveboZJg0qUpmJZb5VqzN" crossorigin="anonymous" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
```
It would be nice to have no third party requests, especially to these hosts.

Commit:
add files

Commit:
Merge branch 'master' of https://github.com/rickmort/website

Commit:
add hr

Commit:
add main section

Commit:
eliminate uppercase

Commit:
create about section

Commit:
change about section background color

Commit:
add css about styles

Commit:
back design and fix dimensions

Commit:
eliminate unnecessary images

Commit:
eliminate pages acces

the previous access to these pages are no longer occupied, everything is now on the landing page.

Commit:
update about text content

Commit:
update main text content

Commit:
Update config.toml

Commit:
justify chat section

Commit:
justify about text

Commit:
upadate headers

Commit:
update

Commit:
docker file

Commit:
Chage base url https://planetdecred.org/
